README.md

```diff
@@ -120,7 +119,7 @@ Implemented compiler passes:
 
 Goals:
 
-  * Have very small binary sizes. Don't pay for what you don't use.
+  * Have very small binary sizes. Only pay for what you use.
```

Comment:
"Don't pay for what you don't use" is a reference to a C++ motto, so I would keep that one intact.